introduction.md

```diff
@@ -1,3 +1,9 @@
 Hi, I'm Victoria
 
 My favorite path so far is JavaScript
+
+My skills are:
+
+* JavaScript
+
+* Git
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?